build/x86_64_jessie/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:jessie
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -y && apt-get install -y \
+        autoconf \
```

Comment:
The level of indentation is inconsistent between debian and centos. Can we stick with 2 indentations per level?

Reply:
Done.